snafu/stressng_wrapper/trigger_stressng.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import subprocess
+import time
+import logging
+import yaml
+
+logger = logging.getLogger("snafu")
+
+class Trigger_stressng():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.runtype = args.runtype
+        self.timeout = args.timeout
+        self.vm_stressors = args.vm_stressors
+        self.vm_bytes = args.vm_bytes
+        self.mem_stressors = args.mem_stressors
+
+    def _run_stressng(self):
+        cmd = "stress-ng --job /workload/jobfile --log-file stressng.lg -Y stressng.yml"
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        stdout, stderr = process.communicate()
+        return stdout.strip().decode("utf-8"), process.returncode
+
+    def _parse_outfile(self):
+        stream = open('stressng.yml', 'r')
+        data = yaml.load(stream, Loader=yaml.FullLoader)
+        metrics = (data["metrics"])
+        results = []
+        for metric in metrics:
+            stressor = (metric["stressor"])
+            bogoops = (metric["bogo-ops"])
+            result = {stressor : bogoops}
+            print(result)
```

Comment:
Remove this print, we're already printing metrics with logger.info(data)

Reply:
done